api/views/course.py

```diff
@@ -206,7 +210,7 @@ def my_grades(self, request, pk):
         return Response(results)
 
     @action(detail=True, methods=["get"], url_path="grade-book", permission_classes=[GradeBookPermission])
-    def course_grade_book(self, request, pk):
+    def course_grade_book(self, request, pk, response=True):
```

Comment:
you should not use a view and a function that returns data. You should extract the getting grade book logic to another file (probably in utils or services folder of the canvas app). And use that function in both views.

Reply:
I extracted it to the canvas services folder since use case seemed similar to the stats functions in the event file located there. I am curious if we have a specific system for where we are placing things between the different folders (api vs canvas vs course) and services vs utils